tasks/postgres-database-extensions.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: "Install required packages for 'postgres_ext' ansible module"
+  apt:
+    name: "{{ item }}"
+    state: present
+  loop:
+    - libpq-dev
+    - python-psycopg2
+
+- name: Install system dependencies of extensions
+  apt:
+    name: "{{ item }}"
+    state: present
+    cache_valid_time: 86400
+  loop: "{{ postgres_extensions.apt_deps | default([]) }}"
```

Comment:
Isn't a postgresql restart needed before being able to issue a `CREATE EXTENSION` ?

Reply:
I need to try that because I do not want to add a "restart" handler in the this role (too dangerous for a production database)

Reply:
Running the tests (that I had to fix btw...) **without* a PG restart seems to create the extension well:
```
TASK [ansible-postgresql-role : Create database extensions] **************************
included: /ansible-postgresql-role/tasks/postgres-database-extensions.yml for postgres_one                                                                  

TASK [ansible-postgresql-role : Install required packages for 'postgres_ext' ansible module] ***                                                                            
ok: [postgres_one]

TASK [ansible-postgresql-role : Install system dependencies of extensions] ***********
changed: [postgres_one]

TASK [ansible-postgresql-role : CREATE EXTENSION on database test_database] **********
changed: [postgres_one] => (item=postgis)
changed: [postgres_one] => (item=postgis_topology)
changed: [postgres_one] => (item=btree_gist)
```

So this should work without a PG restart :)